articles/cognitive-services/Face/Tutorials/FaceAPIinJavaForAndroidTutorial.md

```diff
@@ -289,13 +304,15 @@ Finally, add a call to the detectAndFrame method from the onActivityResult metho
                 ImageView imageView = (ImageView) findViewById(R.id.imageView1);
                 imageView.setImageBitmap(bitmap);
      
-                **detectAndFrame(bitmap);**
+     		// uncomment the following:
```

Comment:
1. Indentation.
2. Comments should be a complete sentence, e.g. start with Uppercase and end with a proper punctuation.
3. I prefer the comments should be something like `This is the new addition.`